internal/pkg/utils/network_linux.go

```diff
@@ -187,20 +187,33 @@ func DoNetworking(
 				return fmt.Errorf("failed to set net.ipv6.conf.lo.disable_ipv6=0: %s", err)
 			}
 
-			// No need to add a dummy next hop route as the host veth device will already have an IPv6
-			// link local address that can be used as a next hop.
-			// Just fetch the address of the host end of the veth and use it as the next hop.
-			addresses, err := netlink.AddrList(hostVeth, netlink.FAMILY_V6)
-			if err != nil {
-				logger.Errorf("Error listing IPv6 addresses for the host side of the veth pair: %s", err)
-				return err
+			var err error
```

Comment:
Please can you add a comment explaining why the retry is needed?   Something like `// Sometimes the IPv6 address takes a short time to appear after the sysctls are enabled.  Retry a few times.`

Reply:
added :)